src/test/java/echoServer/MockServerSocketWrapperTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package echoserver;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import static org.junit.Assert.*;
```

Comment:
I think it's generally better not to use * to import if there are just a small number of things to import. I think around 5 import statements or so is when I'd rather use the *. What do you two think?

Reply:
I agree. There's no reason to import the whole thing if we're only using a few of the classes/packages.